pkg/operator/etcdendpointscontroller/etcdendpointscontroller.go

```diff
@@ -111,6 +112,39 @@ func (c *EtcdEndpointsController) syncConfigMap(ctx context.Context, recorder ev
 		}
 	} else if !errors.IsNotFound(err) {
 		klog.Warningf("required configmap %s/%s will be created because it was missing: %w", operatorclient.TargetNamespace, "etcd-endpoints", err)
+	} else if existing == nil {
+		// create endpoint addresses for each node
+		nodes, err := c.nodeLister.List(labels.Set{"node-role.kubernetes.io/master": ""}.AsSelector())
+		if err != nil {
+			return fmt.Errorf("unable to list expected etcd member nodes: %v", err)
+		}
+		endpointAddresses := map[string]string{}
+		for _, node := range nodes {
+			var nodeInternalIP string
+			for _, nodeAddress := range node.Status.Addresses {
+				if nodeAddress.Type == corev1.NodeInternalIP {
+					nodeInternalIP = nodeAddress.Address
+					break
+				}
+			}
+			if len(nodeInternalIP) == 0 {
+				return fmt.Errorf("unable to determine internal ip address for node %s", node.Name)
+			}
+			endpointAddresses[node.Name] = nodeInternalIP
```

Comment:
@hasbro17  @dusk125  I avoided the `base64` encoding to pass the test, do not know if this is correct way !!